Units/cpp-override.d/input.cpp

```diff
@@ -0,0 +1,18 @@
+class Base
+{
+public:
+	virtual foo() = 0;
```

Comment:
BTW, it would be better to have valid C++ code here, e.g. give a type to all these declarations.